NYPL/dgx-header-component
# SCC-3227: Adding a third radio button for searching - React16 version

Resolves [SCC-3227](https://jira.nypl.org/browse/SCC-3227)

- Essentially the same work update as PR #129 
- This adds the infrastructure code to enable a third radio button for the search form.
- The current mobile search dropdown contains two buttons. This is now updated to include the three radio buttons instead of adding another button. This can be compared on the prod site https://header.nypl.org/.
- This PR points to a feature branch called `third-radio-btn-feature-reactupdate`. This feature won't be released soon so we don't want to merge it into the React16 branch `reactupdate`.
- The verbiage isn't complete and its addition will be a separate PR.
- ~The URLs for searching aren't complete and their addition will be a separate PR. This means that when searching and selecting "Search the Circulating Catalog" and "Search the Research Catalog", both selections will go to the same site.~ This is no longer the case and selecting "Search the Research Catalog" will go to the Research Catalog site.

## How to test

- Clone the repo and install dependencies
- Make sure you are using node v10
- Run `npm run start:dev` and it should open the dev site in a browser at "localhost:8080".
- Test on mobile and desktop.
